DNS_CONFIGURATION.md

```
@@ -0,0 +1,331 @@
+
+```bash
+# 使用相同流程，但指向 staging ALB
+# 修改 Ingress host 为 api-staging.nova.social
+```
+
+### Development 环境
+
+域名: `api-dev.nova.social`
+
+或者使用 IP + nip.io:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)